client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -0,0 +1,351 @@
+import os
+import sys
+import time
+import stat
+import shutil
+
+import king_phisher.client.plugins as plugins
+
+from gi.repository import Gtk
+from gi.repository.GdkPixbuf import Pixbuf
+
+import boltons.strutils
+import boltons.timeutils
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Josh Jacob']
+	title = 'SFTP Client GUI'
+	description = """
+	How many foos could fu foo is fu fooed fu foos on foo.
+	"""
+	homepage = 'https://github.com/securestate/king-phisher'
+
+	def initialize(self):
+		module = sys.modules['king_phisher.client.application']
+		self.temp = module.KingPhisherClientApplication.do_sftp_client_start
+		module.KingPhisherClientApplication.do_sftp_client_start = self.new_SFTP_signal
+		sys.modules['king_phisher.client.application'] = module
+		return True
+
+	def _cleanup(self):
+		module = sys.modules['king_phisher.client.application']
+		module.KingPhisherClientApplication.do_sftp_client_start = self.temp
+		sys.modules['king_phisher.client.application'] = module
+
+	def new_SFTP_signal(self):
+		if not self.application._ssh_forwarder:
```

Comment:
That could be because the signal is not being properly connected. After implementing my feedback in regards to that, revisit this condition. If you can't figure it out let me know and I'll take a look.